generators/server/prompts.js

```diff
@@ -188,6 +188,10 @@ function askForServerSideOpts(meta) {
                     name: 'MongoDB'
                 },
                 {
+                    value: 'couchbase',
+                    name: 'Couchbase'
```

Comment:
It could be prudent to mention it's a beta version at first: `[BETA] Couchbase` as it is done with other newly added technologies. @jdubois what do you think?

Reply:
Yes for a first release, I'd rather have "[BETA]" for a few months